subsys/nfc/t4t/isodep.c

```diff
@@ -269,6 +269,10 @@ static int ats_parse(const uint8_t *data, size_t len)
 	/* Check if ATS contains historical bytes. */
 	if (index < len) {
 		t4t_isodep.tag.historical_len = len - index;
+		if (t4t_isodep.tag.historical_len > NFC_T4T_ISODEP_HIST_MAX_LEN) {
```

Comment:
Please add a comment saying that parsing is limited to 15 bytes as specification mandates. Parsing of further historical bytes is not supported.